src/08-accessing.js

```diff
@@ -171,7 +192,21 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
```

Comment:
This is not exactly hardcoded but it can definitely be refactored to be more dynamic. How can you use a loop to create the flat object?